src/couch_mrview_util.erl

```diff
@@ -1040,6 +1042,16 @@ slice_dbcopy([Name|Names], AllDbCopies, Acc0) ->
 
 %% End of <= 1.2.x upgrade code.
 
+extract(counts, Red) when is_tuple(Red) ->
```

Comment:
Would you mind swapping the order so `user_reds` clause goes before `counts` clause. It looks like `user_reds` case would be called more often.